test/json/any_json_helper.cpp

```diff
@@ -70,7 +70,7 @@ struct any_json_helper_t {
     static const T& as(const value_type& x) { return any_cast<const T&>(x); }
         
     static void move_append(object_type& obj, key_type& key, value_type& value) {
-        obj.emplace(move(key), move(value));
+        obj.emplace(std::move(key), std::move(value));
```

Comment:
I'm not exactly sure, since I don't remember the exact error text, but this was motivated by a specific error.

I imagine this is because of the `using namespace std;` and `using namespace boost;` above, combined with the fact that boost and the standard, both don't give guarantees about which files are included by other files.